hw03_ramav.py

```diff
@@ -0,0 +1,128 @@
+#################################################################################
+# Author:Valentin Rama
+# Username: ramav
+#
+# Assignment: HW03
+# Purpose: Creating a cat
+# Google Doc Link: https://docs.google.com/document/d/18V8i2I56Gj5T8IJGo4s0cEIvQE2YgXbE2uGVO0n7EVE/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+# TESTING PUSH FEATURE IN TA HOURS
+
+import turtle
+
+def draw_ears(cat_turtle):
+    """
+    The function draw_ears allows for the drawing of the cat to draw the ears. using goto allows for the ears
+    to be drawn at a certain position that allows for the ears to be drawn. The fillcolor allows so the spce where the
+    ears are drawn they are filled with the color "gray" in this instance.
+    """
+
+    cat_turtle.penup()
+    cat_turtle.goto(-25, 100)
+    cat_turtle.pendown()
+    cat_turtle.fillcolor("gray")
+    cat_turtle.begin_fill()
+    cat_turtle.forward(20)
+    cat_turtle.right(90)
+    cat_turtle.forward(20)
+    cat_turtle.left(45)
+    cat_turtle.end_fill()
+
+    cat_turtle.penup()
+    cat_turtle.goto(0, 100)
+    cat_turtle.pendown()
+    cat_turtle.fillcolor("gray")
+    cat_turtle.begin_fill()
+    cat_turtle.left(45)
+    cat_turtle.forward(20)
+    cat_turtle.right(90)
+    cat_turtle.forward(20)
+    cat_turtle.left(45)
+    cat_turtle.end_fill()
+
+def draw_cat(cat_turtle):
+
```

Comment:
Remove blank lines after function definitions (the docstring should always be directly after the `def` line).